src/tools/gcc.lua

```diff
@@ -481,6 +489,13 @@
 			table.insert(flags, '-L' .. p.quoted(dir))
 		end
 
+		if (table.contains(os.getSystemTags(cfg.system), "darwin")
+			and cfg.frameworkdirs ~= nil and #cfg.frameworkdirs) then
```

Comment:
This should be on one line. Also, this is not consistent with the other `dirs` blocks by checking the value of `cfg.frameworkdirs`.

Reply:
Ok for the "one line". Here, `cfg.frameworkdirs` is always valid so I will remove the unnecessary tests